polyfill/lib/ecmascript.mjs

```diff
@@ -2306,13 +2306,13 @@ export function DisambiguatePossibleInstants(possibleInstants, timeZone, dateTim
   throw new Error(`assertion failed: invalid disambiguation value ${disambiguation}`);
 }
 
-export function GetPossibleInstantsFor(timeZone, dateTime) {
+export function GetPossibleInstantsFor(timeZone, dateTime, getPossibleInstantsFor = undefined) {
   if (typeof timeZone === 'string') {
     const TemporalTimeZone = GetIntrinsic('%Temporal.TimeZone%');
     timeZone = new TemporalTimeZone(timeZone);
     return Call(GetIntrinsic('%Temporal.TimeZone.prototype.getPossibleInstantsFor%'), timeZone, [dateTime]);
   }
-  const getPossibleInstantsFor = GetMethod(timeZone, 'getPossibleInstantsFor');
+  getPossibleInstantsFor ??= GetMethod(timeZone, 'getPossibleInstantsFor');
```

Comment:
Is this comparing to `undefined` in the spec? If yes, then should we compare the same here, instead of `??=` that also picks up `null`?

Reply:
This seemed fine to me because it's an internal function - we don't get this value from a user API.